pyproject.toml

```diff
@@ -72,7 +72,7 @@ inquirerpy = "^0.3.4"
 statsmodels = "^0.13.5"
 umap-learn = "^0.5.3"
 streamlit-plotly-events = "^0.0.6"
-encord-active-components = "^0.0.5"
+encord-active-components = { path="./frontend_components", develop=true}
```

Comment:
**TODO**: update me to the latest after the PR has been reviewed (to only publish the final changes)